eng/pipelines/templates/variables/globals.yml

```diff
@@ -8,4 +8,7 @@ variables:
   EmulatorMsiUrl: 'https://aka.ms/cosmosdb-emulator'
   AdditionalOptions: ''
   # format !<group1Id>:<artifact1Id>, !<group2Id>:<artifact2Id>,... no trailing comma is necessary
-  PLSkipOptions: ''
\ No newline at end of file
+  PLSkipOptions: ''
+  # This is the default Java version. It's the version used to build the shipping libraries, spot bugs etc.
+  JavaLTSVersion: '1.11'
```

Comment:
As far as I can tell nothing is even using JavaLTSVersion anywhere so should we even keep it?

Reply:
It's gone now.